documentation/modules/ROOT/pages/transformations/timezone-converter.adoc

```diff
@@ -0,0 +1,181 @@
+transforms.convertTimezone.converted.timezone=+05:30
+transforms.convertTimezone.exclude.fields=source:customers:updated_at
+----
+
+[[timezone-converter-configuration-options]]
+== Configuration options
+
+The following table lists the configuration options that you can use with the `TimezoneConverter` SMT.
+
+.TimezoneConverter SMT configuration options
+[cols="14%a,40%a,10%a,10%a"]
+|===
+|Property
+|Description
+|Type
+|Importance
+
+|[[timezone-converter-converted-timezone]]<<timezone-converter-converted-timezone, `converted.timezone`>>
+|A string that specifies the target timezone to which the timestamp fields should be converted. The target timezone can be specified as a geographic timezone e.g. `America/New_York` or as a UTC offset e.g. `+02:00`.
+|string
+|high
+|[[timezone-converter-include-fields]]<<timezone-converter-include-fields, `include.fields`>>
+|A comma-separated list of rules that specify which fields should be included for timezone conversion, using one of the following formats:
+
+    `source:<tablename>` :: Matches Debezium change events with a source information block having the specified table name. All time-based fields in the matched table will be converted.
+    `source:<tablename>:<fieldname>` :: Matches Debezium change events with a source information block having the specified table name. Only the specified field name will be converted.
+    `topic:<topicname>` :: Matches events from the specified topic name, converting all time-based fields.
+    `topic:<topicname>:<fieldname>` :: Matches events from the specified topic name, converting only the specified field name.
+    `<matchname>:<fieldname>` :: Applies a heuristic matching algorithm to match against the source information block table name if present; otherwise, matches against the topic name. Conversion is applied only to the specified field name.
+|list
+|medium
+|[[timezone-converter-exclude-fields]]<<timezone-converter-exclude-fields, `exclude.fields`>>
+|A comma-separated list of rules that specify what fields should be excluded from timezone conversion, using one of the following formats:
+
+    `source:<tablename>` :: Matches Debezium change events with a source information block having the specified table name. All time-based fields in the matched table will be excluded from conversion.
+    `source:<tablename>:<fieldname>` :: Matches Debezium change events with a source information block having the specified table name. Only the specified field name will be excluded from conversion.
+    `topic:<topicname>` :: Matches events from the specified topic name, excluding all time-based fields.
+    `topic:<topicname>:<fieldname>` :: Matches events from the specified topic name, excluding only the specified field name.
+    `<matchname>:<fieldname>` :: Applies a heuristic matching algorithm to match against the source information block table name if present; otherwise, matches against the topic name. Excludes conversion only for the specified field name.
```

Comment:
```suggestion
    `<matchname>:<fieldname>` :: Applies a heuristic matching algorithm to match against the against the table name of the source information block, if present; otherwise, matches against the topic name. 
The SMT excludes from conversion only fields that have the specified name.
```